checkoff

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import subprocess
+import sys
+
+from ocflib.account.search import user_exists
+
+
+# Format of JSON file should be:
+# {
+#     "roster": [<list of OCF usernames of students>],
+#     "users": {
+#         "username1": [<list of labs checked off>],
+#         ...
+#     },
+#     "labs": {
+#         "a1": [<list of usernames checked off],
+#         ...
```

Comment:
It's redundant for ease of use and querying. I don't really think it matters whichever way we do it since there aren't many students anyway, so if it really bothers you I can just rewrite it.

Reply:
eh my bad you're right ill just change it lmao